core/conn/jdbc_type2/native/Debug.cpp

```diff
@@ -183,8 +184,9 @@ static const char *UCase(const char *text)
 
 static unsigned long GetThreadId(void)
 {
-	_TSLX_t thread_id = tslx_ext_pthread_self();
-	return((unsigned long) thread_id.field1);
+	//_TSLX_t thread_id = tslx_ext_pthread_self();
+	//return((unsigned long) thread_id.field1);
+        return pthread_self();
```

Comment:
Consider removing TSLX thread and pthread package references in T2 driver. T2 driver may not need to deal with threads at all. Also consider using syscall(SYS_gettid); to get the thread id to avoid pulling in pthread references.

You should feel free to drop all wanted cpp and .h files related to threads